### ISSUE TITLE
signal-cli fetched receipts

### ISSUE THREAD

**Issue by piramiday** [AsamK/signal-cli#284]
my `signal-cli` instance does not seem to send "fetched" receipts back to my primary mobile app.

e.g.:
![Screenshot at 2020-03-26 15-43-04](https://user-images.githubusercontent.com/48775554/77703662-9bde0980-6f78-11ea-9238-9ae28bbe96d5.png)

as opposed to:
![Screenshot at 2020-03-26 15-53-12](https://user-images.githubusercontent.com/48775554/77704278-f4fa6d00-6f79-11ea-8725-7e1bbcb78cb4.png)

I am absolutely positive that it used to work for me, that is, after a `signal receive` I would pass from screenshot 1 to screenshot 2.

am I missing something?
I have recently removed a second identity, and I've tried giving `updateAccount` and resetting sessions.
thanks

**Comment by piramiday** [comment]
@AsamK is 1b56485fc8ce1d82d1e5e8aaeca82bcd39bc7c26 related to this, by any chance?

**Comment by AsamK** [comment]
yes, that commit has fixed this issue.
With which signal-cli version did you test it?

**Comment by piramiday** [comment]
okay, thanks, just checking. hopefully I'll close this issue with the new release!
I am currently running the pre-built latest release, 0.6.5.

I have tried compiling from source and running the newer binary, but my signal failed so I could not test that the problem was solved.
```
$ ./signal-cli/build/install/signal-cli/bin/signal-cli -u +12223334567 receive
Exception in thread "main" java.lang.AssertionError: Must have either a UUID or E164 number!
	at org.whispersystems.signalservice.api.push.SignalServiceAddress.<init>(SignalServiceAddress.java:34)
	at org.whispersystems.signalservice.api.push.SignalServiceAddress.<init>(SignalServiceAddress.java:51)
	at org.whispersystems.signalservice.api.push.SignalServiceAddress.<init>(SignalServiceAddress.java:46)
	at org.asamk.signal.storage.SignalAccount.load(SignalAccount.java:207)
	at org.asamk.signal.storage.SignalAccount.load(SignalAccount.java:75)
	at org.asamk.signal.manager.Manager.init(Manager.java:212)
	at org.asamk.signal.Main.handleCommands(Main.java:108)
	at org.asamk.signal.Main.main(Main.java:60)
```

**Comment by piramiday** [comment]
fixed with 57390ae7358bf0a5d964be515360819e7bb901c0 :+1:
